overlord/snapstate/snapstate_test.go

```diff
@@ -834,6 +834,8 @@ func (s *snapmgrTestSuite) TestByKindOrder(c *C) {
 }
 
 func (s *snapmgrTestSuite) TestUpdateManyWaitForBases(c *C) {
```

Comment:
maybe having two tests for the 16 vs 18 case would be better?

Reply:
Done now.